### DISCUSSION TITLE
Adding icon for all platforms

### DISCUSSION THREAD

**Discussion by TheRobo-13** [unoplatform/uno#8029] [Q&A]
How do I add an app icon for all platforms (pc, mobile, web, etc)?

**Comment by jeromelaban** [comment]
Can you specify which icon you're talking about? Some targets don't have that notion.

**Reply by TheRobo-13** [reply]
I'm trying to get the app to have an icon for Mobile

**Reply by jeromelaban** [reply]
For mobile, see this: https://docs.microsoft.com/en-us/xamarin/ios/app-fundamentals/images-icons/app-icons and https://docs.microsoft.com/en-us/xamarin/android/deploy-test/release-prep/?tabs=windows

**Reply by carlh98** [reply]
Hi @jeromelaban, with dotnet6, I don't have access to the Apple section
![image](https://github.com/unoplatform/uno/assets/7770915/828d4005-2457-4156-85fe-8c4c196ade8a)

I would like to add an Asset Catalog, do you have an idea on this?
Previously, this is what we had with a Xamarin projet
![image](https://github.com/unoplatform/uno/assets/7770915/e4aa4997-49f5-417d-a73f-4d089c7f157c)

If needed I can create a new discussion for dotnet6.

**Reply by jeromelaban** [reply]
This is visual studio issue to be opened as a visual studio feedback, as MAUI apps have the same behavior. Uno should not be providing those templates.

**Reply by carlh98** [reply]
Got it, thank you
